.eslintrc.js

```diff
@@ -22,10 +22,12 @@ module.exports = {
     },
   },
   rules: {
+  	"no-bitwise": "off",
     "comma-dangle": ["error", "never"],
     "object-curly-spacing": ["error", "always"],
     "quotes": ["error", "single",  { "avoidEscape": true }],
     "no-unused-vars": ["error", { "args": "none" }],
+    "react-hooks/exhaustive-deps": "off",
```

Comment:
Disable this link because most of time we use it as componentDidMounted and cleanup when componentWillUnmounted, so I do 
```javascript
useEffect(()=> {
  //do something
}, []) //empty array here!
```

but yarn will always give an error says I need to add dependent variable here.
And if you turn it on and run `yarn lint:fix`, you will see lint will add all the related variables into dependent array, which is not the case we want to use it.

Reply:
I don't understand what you mean. In most cases you really shouldn't just ignore your dependent variables for convenience. Eslint is there for a reason. Could you point to me a specific example that justifies this?

Reply:
I look back and through the link you posted, I think I really previously have some misunderstanding about `useEffect`. And this [commnet](https://github.com/facebook/react/issues/14920#issuecomment-470945051) is especially useful. 
Enable the lint again.

Reply:
cool cool